lmp/build.sh

```diff
@@ -36,7 +37,7 @@ cp /root/.netrc /home/builder/.netrc || true
 
 set_base_lmp_version
 
-mkdir build conf
+mkdir -p build conf
```

Comment:
-p forces to create mutli-level subdirs. There are 2 separate 1-level subdirs. I doubt we need there `-p`

with this - lgtm

Reply:
it also won't err out if the directory exists. I think he's doing this instead `[ -d build ] || mkdir build`.

Reply:
> it also won't err out if the directory exists. I think he's doing this instead [ -d build ] || mkdir build.

yes that was really the reason

Reply:
> it also won't err out if the directory exists. I think he's doing this instead `[ -d build ] || mkdir build`.

Oh, suck a good trick, didn't aware of this side-effect :)